objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/neuralNet/YoloProcessor.java

```diff
@@ -108,11 +120,25 @@ private void setup(Map<String, ?> neuralNet, String modelDirectory) throws Excep
        }
               
        // Setup the variables for saving images
-       if (neuralNet.get("outputDir") instanceof String) {
-           outputDir = (String) neuralNet.get("outputDir");
+       if (neuralNet.get("saveImage") instanceof String) {
```

Comment:
if not instance of String?

If already checked, then doesn't seem like we have to check here.  If not already checked, we shouldn't silently ignore.